webview/src/shared/components/slider/MinMaxSlider.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FormEvent } from 'react'
+import styles from './styles.module.scss'
+
+interface MinMaxSliderProps {
+  minimum?: number
+  maximum: number
+  step?: number
+  defaultValue: number
+  label: string
+  onChange: (newValue: number) => void
+}
+
+export const MinMaxSlider: React.FC<MinMaxSliderProps> = ({
+  minimum,
```

Comment:
If the screen reaches a certain small width, the slider can't move

https://user-images.githubusercontent.com/43496356/223740405-ac7e2990-8780-4f3a-99f9-8b0f117775bd.mov

Should we hide/disable the slider in this case?